### DISCUSSION TITLE
why Openradioss takes more time for executing the same model on windows as compared to linux (without mpi).

### DISCUSSION THREAD

**Discussion by ShobhaKumari07** [OpenRadioss/OpenRadioss#2878] [Show and tell]
I ran openradioss on windows as well as Linux.  on windows it is taking lot more time as compared to Linux, why??

**Comment by laurent-altr** [comment]
Hi,
could you tell us more about the hardware you are using?  Also could you provide some examples? Ideally the *1.out for the same model on windows and Linux. 

Note that we are using different compilers for Linux and windows, but I don't expect a big difference between both. (15% difference in one way or the other, depending on the model).  

Thank you, 
Laurent.
